hordes-ui-mod.js

```diff
@@ -18,36 +18,45 @@
   * TODO: (MAYBE, Confirm if dev is ok w it) Ctrl clicking item in inventory when Merchant is open to paste item name into search field, and search
   * TODO: (MAYBE, confirm if dev is ok w it) Ctrl clicking item to copy details so user can paste in chat
   */
-(function() {
+(function () {
     'use strict';
 
     // If this version is different from the user's stored state,
     // e.g. they have upgraded the version of this script and there are breaking changes,
     // then their stored state will be deleted.
     const BREAKING_VERSION = 1;
-    const VERSION = '0.190'; // Should match version in UserScript description
+    const VERSION = '0.201'; // Should match version in UserScript description
```

Comment:
0.200 is fine, this version is for the public so going from .190 -> .200 makes sense.